libnmstate/nm/sriov.py

```diff
@@ -43,11 +90,54 @@ def create_setting(iface_state, base_con_profile):
         if not sriov_setting:
             sriov_setting = nmclient.NM.SettingSriov.new()
 
+        vfs_config = sriov_config.get(Ethernet.SRIOV.VFS_SUBTREE, [])
+        vf_object_ids = {vf.get_index() for vf in sriov_setting.props.vfs}
+        vf_config_ids = {
+            vf_config[Ethernet.SRIOV.VFS.ID] for vf_config in vfs_config
+        }
+
+        # As the user must do full edit of vfs, nmstate is deleting all the vfs
```

Comment:
Please specify in the commit message this limitation: "Currently, only full editation of VF/s is supports. Any VF/s that are emitted from the desired state are therefore implicitly initialized."

Reply:
Done, thank you!